deploy.sh

```diff
@@ -152,8 +161,12 @@ svn update
 
 svn status
 
-echo "➤ Committing files..."
-svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+if [[ "$DRY_RUN" == true ]]; then
+  echo "➤ Debug mode: Files not committed."
+else
+  echo "➤ Committing files..."
+  svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
```

Comment:
Right now this is the only step we skip if running in dry mode. Wondering if we should also skip some of the other `svn` steps, like `svn add . --force > /dev/null` and `svn rm %@ > /dev/null` and `svn propset`. Seems like there's no need to run those if we aren't actually committing, though maybe no downside to running those?

Reply:
It could be useful if we decide to do some kind of "local" commit in a dry run context in order to provide more feedback as to what would actually happen on the main repository which is what I think @joemcgill may have been thinking. For the most part though, I think we probably want to make the dry run as close to the real thing as possible otherwise we risk losing the benefit of doing a dry run in the first place.

Reply:
Agree with @aaemnnosttv